tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -26,6 +27,109 @@
 from openvino.tools.mo.utils.utils import refer_to_faq_msg, get_mo_root_dir
 from openvino.tools.mo.utils.version import get_version
 
+cli_tool_specific_descriptions = {
+    'input_model':
+        'Tensorflow*: a file with a pre-trained model '
+        '(binary or text .pb file after freezing). '
+        'Caffe*: a model proto file with model weights.',
+    'input_shape':
+        'Input shape(s) that should be fed to an input node(s) '
+        'of the model. Shape is defined as a comma-separated '
+        'list of integer numbers enclosed in parentheses or '
+        'square brackets, for example [1,3,227,227] or '
+        '(1,227,227,3), where the order of dimensions depends '
+        'on the framework input layout of the model. For '
+        'example, [N,C,H,W] is used for ONNX* models and '
+        '[N,H,W,C] for TensorFlow* models. The shape can '
+        'contain undefined dimensions (? or -1) and should fit '
+        'the dimensions defined in the input operation of the '
+        'graph. Boundaries of undefined dimension can be '
+        'specified with ellipsis, for example '
+        '[1,1..10,128,128]. One boundary can be undefined, for '
+        'example [1,..100] or [1,3,1..,1..]. If there are '
+        'multiple inputs in the model, --input_shape should '
+        'contain definition of shape for each input separated '
+        'by a comma, for example: [1,3,227,227],[2,4] for a '
+        'model with two inputs with 4D and 2D shapes. '
+        'Alternatively, specify shapes with the --input option.',
+    'input':
```

Comment:
May be we should have all of this documentation in one place in separate .py file. And ti should be under some function like `get_help_message(is_cli_tool: bool)`. Depending on the flag `is_cli_tool` we return help message for `convert_model` or tool.
It will help to avoid duplication